readme.txt

```diff
@@ -28,6 +31,11 @@ Also please check out our other [plugins](https://draftpress.com/products/?utm_s
 2. Select your image size when inserting image into a post
 
 == Changelog ==
+= 1.2.9 = 2023-21-03
```

Comment:
@KZeni can you please change the changelog date to today's date? then we can accept this PR.

Reply:
Will update the commit accordingly.

Btw, I'm going to use WordPress 6.3 for the new tested up to value as that's just around the corner with this looking fine on top of this plugin being fairly straightforward with 6.3 then not really making updates that'd directly affect this plugin.

Also, FWIW, saying something like tested up to WP version x.x automatically has any/all subsequent x.x.x version releases also say they're supported since those don't/shouldn't introduce any features/changes that would affect plugin functionality like x.x updates could/do (so specifying those minor versions is just unnecessary & only gets in the way of things.) https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#readme-header-information has this officially documented per:
>**Tested up to** – The version of WordPress that the plugin has been tested against. _This field ignores minor versions_, as plugins shouldn’t break with a minor update. _This means a plugin only needs to define the major version it is tested against and the WordPress.org plugin directory will automatically add the minor version_. This should be numbers only, so ‘4.9’ and not ‘WP 4.9’